tools/espota.py

```diff
@@ -7,9 +7,9 @@
 # Modified since 2015-11-09 from Hristo Gochkov (https://github.com/me-no-dev)
 #
 # This script will push an OTA update to the ESP
-# use it like: python espota.py -i <ESP_IP_address> -p <ESP_port> [-a password] -f <sketch.bin>
+# use it like: python espota.py -i <ESP_IP_address> -I <Host_interface> -p <ESP_port> -P <Host_port> [-a password] -f <sketch.bin>
 # Or to upload SPIFFS image:
-# python espota.py -i <ESP_IP_address> -p <ESP_port> [-a password] -s -f <spiffs.bin>
+# python espota.py -i <ESP_IP_address> -I <Host_interface> -p <ESP_port> -P <HOST_port> [-a password] -s -f <spiffs.bin>
```

Comment:
This should probably be <host IP>, because <host interface> might imply something like eth0 or wlan0.